pkg/html/html.go

```diff
@@ -727,6 +731,188 @@ func testImpactingComponents(testImpactingBugs []bugsv1.Bug) string {
+					<button class="btn btn-primary btn-sm py-0" style="font-size: 0.8em" type="button" data-toggle="collapse" data-target=".%[4]s" aria-expanded="false" aria-controls="%[4]s">Expand Failing Tests</button>
+				</td>
+				<td>
+					%0.2f%% <span class="text-nowrap">(%d/%d failed/total)</span>
+				</td>
+				<td/>
+				<td>
+					NA
+				</td>
+			</tr>
+		`
+
+	for _, v := range failuresByBugzillaComponent {
+		safeBZJob := fmt.Sprintf("%s---component", v.Name)
+		safeBZJob = strings.ReplaceAll(safeBZJob, ".", "")
+		safeBZJob = strings.ReplaceAll(safeBZJob, " ", "")
+
+		prev := util.GetPrevBugzillaJobFailures(v.Name, failuresByBugzillaComponentPrev)
+		highestFailPercentage := v.JobsFailed[0].FailPercentage
+		lowestPassPercentage := 100 - highestFailPercentage
+		rowColor := ""
+		switch {
+		case lowestPassPercentage > 85:
+			rowColor = "table-success"
+		case lowestPassPercentage > 70:
+			rowColor = "table-warning"
+		case lowestPassPercentage > 0:
+			rowColor = "table-danger"
+		default:
+			rowColor = "error"
+		}
+
+		if prev != nil && len(prev.JobsFailed) > 0 {
+			previousHighestFailPercentage := prev.JobsFailed[0].FailPercentage
+			previousLowestPassPercentage := 100 - previousHighestFailPercentage
+			arrow := ""
+
+			delta := 5.0
+			if v.JobsFailed[0].TotalRuns > 80 {
+				delta = 2
+			}
+			if lowestPassPercentage > previousLowestPassPercentage+delta {
+				arrow = up
+			} else if lowestPassPercentage < previousLowestPassPercentage-delta {
+				arrow = down
+			}
+
+			if lowestPassPercentage > previousLowestPassPercentage+delta {
+				arrow = fmt.Sprintf(up, lowestPassPercentage-previousLowestPassPercentage)
+			} else if lowestPassPercentage < previousLowestPassPercentage-delta {
+				arrow = fmt.Sprintf(down, previousLowestPassPercentage-lowestPassPercentage)
+			} else if lowestPassPercentage > previousLowestPassPercentage {
+				arrow = fmt.Sprintf(flatup, lowestPassPercentage-previousLowestPassPercentage)
+			} else {
+				arrow = fmt.Sprintf(flatdown, previousLowestPassPercentage-lowestPassPercentage)
+			}
+
+			s = s + fmt.Sprintf(bzGroupTemplate,
+				rowColor,
+				v.Name,
+				safeBZJob,
+				lowestPassPercentage,
+				prev.JobsFailed[0].TotalRuns,
+				arrow,
+				previousLowestPassPercentage,
+				prev.JobsFailed[0].TotalRuns,
+			)
+		} else {
+			s = s + fmt.Sprintf(naTemplate,
+				rowColor,
+				v.Name,
+				safeBZJob,
+				lowestPassPercentage,
+				v.JobsFailed[0].TotalRuns,
+			)
+		}
+
+		count := 0
+		for _, failingJob := range v.JobsFailed {
+			bzJobTuple := fmt.Sprintf("%s---%s", v.Name, failingJob.JobName)
+			bzJobTuple = strings.ReplaceAll(bzJobTuple, ".", "")
+			bzJobTuple = strings.ReplaceAll(bzJobTuple, " ", "")
+
+			// only show the worst ten jobs
+			if count > 10 {
+				break
+			}
+			count++
+			s = s + fmt.Sprintf(bzFailingJobTemplate,
+				safeBZJob,
+				v.Name,
+				failingJob.JobName,
+				bzJobTuple,
+				100.0-failingJob.FailPercentage,
+				failingJob.NumberOfJobRunsFailed,
```

Comment:
since you inverted the percentage to be "passing" i think you should invert this to be successes(well, non-failures) also.